FP_analysis_of_reviews_from_Amazon/nearest_review.py

```diff
@@ -25,85 +25,88 @@ def prepare_data(data):
 
 
 def count_deltatime(data):
-    return min(
-        (data[i].time - data[i - 1].time, data[i].text, data[i - 1].text)
-        for i in range(1, len(data))
-    )
+    deltatime = []
+    for i in range(1, len(data)):
```

Comment:
I would iterate over `zip(data[:-1], data[1:])` to avoid indices but it's a matter of personal choice

Reply:
I don't know how I prefer to do, so I tried with zip. All managed.